tests/release/pipelines/fbc_release.go

```diff
@@ -88,7 +88,7 @@ var _ = framework.ReleasePipelinesSuiteDescribe("FBC e2e-tests", Label("release-
 
 			createFBCReleasePlanAdmission(fbcReleasePlanAdmissionName, *managedFw, devNamespace, managedNamespace, fbcApplicationName, fbcEnterpriseContractPolicyName, relSvcCatalogPathInRepo, "false", "", "", "", "")
 
-			component = releasecommon.CreateComponent(*devFw, devNamespace, fbcApplicationName, fbcComponentName, fbcSourceGitURL, "", fbcDockerFilePath, constants.DefaultDockerBuildPipelineBundle)
+			component = releasecommon.CreateComponent(*devFw, devNamespace, fbcApplicationName, fbcComponentName, fbcSourceGitURL, "", fbcDockerFilePath, constants.DefaultFbcBuilderPipelineBundle)
```

Comment:
I tried to run the tests with the change here, seems it didn't work.  In the e2e-tests checking of the PR, the FBC test cases are not included by default since it's disabled [here](https://github.com/konflux-ci/e2e-tests/blob/main/magefiles/magefile.go#L313C112-L313C132). So, if you want to test the change in this PR's e2e-tests checking,  the magefile need to be changed with removing "!" before "release-pipelines".  After the change is verified, restore it back before the PR is merged.

Reply:
let me test it.